.prettierignore

```diff
@@ -1,7 +1,13 @@
+# Automatically regenerated Markdown files based on git patches.
+Toolchain/Patches
+Ports/*/patches
+
 Base/home/anon/Source/js
 Userland/Libraries/LibJS/Tests/invalid-lhs-in-assignment.js
 Userland/Libraries/LibJS/Tests/unicode-identifier-escape.js
 Userland/Libraries/LibJS/Tests/modules/failing.mjs
+# FIXME: Uses nonstandard regex flags which cause a syntax error in prettier.
```

Comment:
Just slap a `// prettier-ignore` on line 8.
`v` _is_ a standard flag (https://tc39.es/ecma262/#sec-get-regexp.prototype.unicodesets)

Reply:
I'm surprised prettier is so bad at this

Reply:
This doesn't work, since prettier fails to compile the file at all with this flag present. I'll change the comment though.